src/CoherenceBuild/Program.cs

```diff
@@ -30,11 +44,17 @@ static int Main(string[] args)
             var outputPath = app.Option("--output-path", "Output path", CommandOptionType.SingleValue);
             var nugetPublishFeed = app.Option("--nuget-publish-feed", "Feed to push packages to", CommandOptionType.SingleValue);
             var apiKey = app.Option("--apikey", "NuGet API Key", CommandOptionType.SingleValue);
-            var ciVolatileShare = app.Option("--ci-volatile-share", "CI Volatile share", CommandOptionType.SingleValue);
```

Comment:
This makes it harder to test this locally (since it'll attempt to point to the CI share which has no write access). Can we undo this?